src/main/java/org/example/PersonTest.java

```diff
@@ -0,0 +1,59 @@
+package org.example;
+
+import org.testng.annotations.Test;
+import static org.testng.Assert.*;
+
+public class PersonTest extends Person{
+
+    public PersonTest(String firstName, String lastName, int age) {
+        super(firstName, lastName, age);
+    }
+
+    @Test (priority=1, description =" Verify if man is retired")
+    public void testIsRetiredForMan() {
+        Man man = new Man("Igor", "Rymaruk", 40);
+        assertFalse(man.isRetired());
+    }
+
+    @Test (priority=2, description = "Verify if woman is retired")
+    public void testIsRetiredForWoman() {
+        Woman woman = new Woman("Oksana", "Shpynta", 55);
+        assertTrue(woman.isRetired());
```

Comment:
why assert is true if requirement says that for woman retaired age starts from 60?